scss/tech.scss

```diff
@@ -1,4 +1,7 @@
 .tech-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
margin in each `tech` should be enough. you can reduce the 1280px width in the container if we want less items per row